pkg/handler/v2/cluster_template/cluster_template_test.go

```diff
@@ -577,3 +577,57 @@ func TestGetClusterTemplateInstance(t *testing.T) {
 		})
 	}
 }
+
+func TestListClusterTemplateInstances(t *testing.T) {
+	t.Parallel()
+	testcases := []struct {
+		Name                   string
+		ExpectedResponse       string
+		TemplateID             string
+		HTTPStatus             int
+		ExistingAPIUser        *apiv1.User
+		ExistingKubermaticObjs []ctrlruntimeclient.Object
+	}{
+		// scenario 1
+		{
+			Name:             "scenario 1: get template instance for global template",
+			TemplateID:       "ctID2",
+			ExpectedResponse: `[{"name":"","spec":{"projectID":"","clusterTemplateID":"","clusterTemplateName":"","replicas":0}},{"name":"my-first-project-ID-ctID2","spec":{"projectID":"my-first-project-ID","clusterTemplateID":"ctID2","clusterTemplateName":"","replicas":10}}]`,
```

Comment:
why is this empty instance in the response?

Reply:
good point. There were two elements in the list. I copy-paste the first empty element.
I fixed this